build.gradle

```diff
@@ -1,5 +1,5 @@
 group 'uk.ac.ebi.subs'
-version '0.1.1-SNAPSHOT'
+version '0.2.1-SNAPSHOT'
```

Comment:
Is it not 0.2.0-SNAPSHOT?

Reply:
This has commits from several dev branches, so it may have gone 0.1.1 to 0.2.0 to 0.2.1

Reply:
OK